client/e2e/poms/operations.ts

```diff
@@ -25,7 +49,106 @@ export class OperationsPOM {
 
   async urlIsCorrect() {
     const path = this.url;
-    const currentUrl = await this.page.url();
-    await expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+    const currentUrl = this.page.url();
+    expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+  }
+
+  async operationsTableIsVisible() {
```

Comment:
I've got more specific comments about some of these `IsVisible` functions in the actual test file, but in general, if they're just a single assertion that doesn't check specific content, I don't think we need them. If something isn't visible, the test is going to fail anyway when we try to click/type into/interact with the non-visible thing

Reply:
It's not considered good practice to use timeouts to mitigate flakey test issues and to use assertions like this to confirm the page is rendered before moving to the next step. I've added most of these to stop flakey test failures. For example I just commented out all of the `...isVisible` assertions in the add operation test and it failed locally, and I had even more failures running in CI. 

Another issue with flakiness is screenshot testing. We want to add some assertions and confirm that the page is rendered before we take a screenshot.

Reply:
Makes sense! I was hoping playwright would be better about flakiness and timing than cypress and we wouldn't have to do random assertions anymore :'(